Sources/VLCCloudStorageTableViewCell.h

```diff
@@ -15,6 +15,7 @@
 #import <BoxSDK/BoxSDK.h>
 #if TARGET_OS_IOS
 #import "GTLRDrive.h"
+#import <OneDriveSDK/OneDriveSDK.h>
```

Comment:
I'm not sure we want to have all the headers included here.

Wouldn't it be better to have the specific ones we need?

Reply:
@Mikanbu we do that with Google drive and Box here as well

Reply:
but you might be right